File: code/PyWrapper_TKO/py-wrapper/UtilityList.py
#import Element.py

class UtilityList :

	def __init__(self,item,sumIutils=0,sumRutils=0):
		"""

		:param item:
		:param elements:
		:param sumIutils:
		:param sumRutils:
		"""
		self.item = item
		#self.elements = elements
		self.sumIutils = sumIutils
		self.sumRutils = sumRutils

	def addElement(self,element):
		"""
		pass an Element object
		"""
		elements = []
		self.sumIutils += element.iutils
		self.sumRutils += element.rutils
		elements.append(element)

File: code/TKO_CSharp/TKO_CSharp/Models/UtilityList.cs
using System.Collections.Generic;

namespace TKO_CSharp.Models
{
    public class UtilityList
    {
        public int Item { get; set; }
        public int SumIutils { get; set; }
        public int SumRutils { get; set; }
        public List<Element> Elements { get; set; }

		public UtilityList(int item)
		{
			Item = item;
			SumIutils = SumRutils = 0;
			Elements = new List<Element>();
		}

		public void AddElement(Element element)
		{
			SumIutils += element.Iutils;
			SumRutils += element.Rutils;
			Elements.Add(element);
		}
	}
}
